tests/scripts/check_files.py

```diff
@@ -24,7 +24,7 @@
     pass
 
 import scripts_path # pylint: disable=unused-import
-from mbedtls_dev import build_tree
```

Comment:
This file has to be modified to check the files that are in the `framework` submodule as well. It seems we just have to change `bytes_output = subprocess.check_output(['git', 'ls-files', '-z'])` to `bytes_output = subprocess.check_output(['git', 'ls-files', '--recurse-submodules', '-z'])`.